src/live-common-set-supported-currencies.js

```diff
@@ -36,4 +37,5 @@ setSupportedCurrencies([
   "bitcoin_testnet",
   "ethereum_ropsten",
   "cosmos_testnet",
+  "crypto_org_croeseid",
```

Comment:
Could you undo the supported currencies change? That way we can merge the PR content straightaway but with the coin being still "disabled" to the public, so it have no impact. Live team will enable it for all once we want to release it publicly.
you are still able to enable it using the envs `EXPERIMENTAL_CURRENCIES`

Thanks.
We tend to do this in